tests/acceptance/cli_population.robot

```diff
@@ -27,13 +27,13 @@ Cli Should Populate App With Installed Keywords
     ...    XML library with 37 keywords loaded.
     ...    Easter library with 1 keywords loaded.
     ...    Process library with 15 keywords loaded.
-    ...    String library with 31 keywords loaded.
+    ...    String library with 32 keywords loaded.

```

Comment:
These changes in tests are due to robot framework version update?

Reply:
Yes, RFWK has been extended with new keyword and some changes to libdoc:
https://github.com/robotframework/robotframework/issues/3534